Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -417,36 +519,36 @@ - (void)addItems:(NSArray*)items {
  @Notes Only general pasteboard, for other type, it just will return nil.
 */
 - (NSString*)string {
-    if ([_name isEqualToString:UIPasteboardNameGeneral]) {
-        return [UIPasteboard _getStringFromClipboard];
+    if (![_name isEqualToString:UIPasteboardNameGeneral]) {
+        TraceWarning(TAG, L"Only general pasteboard supported");
```

Comment:
...in several more spots below

---
In reply to: [87644010](https://github.com/Microsoft/WinObjC/pull/1358#discussion_r87644010) [](ancestors = 87644010)